lxd/daemon.go

```diff
@@ -573,13 +573,13 @@ func (d *Daemon) Init() error {
 
 	/* Print welcome message */
 	if d.MockMode {
-		shared.LogInfo("LXD is starting in mock mode",
+		shared.LogDebug("LXD is starting in mock mode",
```

Comment:
And same for all the other messages in Init(), That's reasonable information to print out on startup and is very useful to understand what the environment is when we get bug reports.